src/v0/analytics/reports/payment_options.ts

```diff
@@ -52,4 +52,30 @@ export class PaymentOptions {
       }
     })
   }
+
+  meta(query?: PaymentOptionsQuery): Promise<PaymentOptionsResponse> {
```

Comment:
I've seen in other places, that we have a default (undefined) for the case that there is no query. Do we actually not need that?

Reply:
Doesn't the nature of the ? after query causes the default to be undefined?